lib/loadJs.js

```diff
@@ -3,13 +3,17 @@
 var requireFromString = require('require-from-string');
 var readFile = require('./readFile');
 
-module.exports = function (filepath) {
-  return readFile(filepath).then(function (content) {
+module.exports = function (filepath, options) {
+  function parseJsFile(content) {
     if (!content) return null;
 
     return {
       config: requireFromString(content, filepath),
       filepath: filepath,
     };
-  });
+  }
+
+  return (!options.sync)
+    ? readFile(filepath).then(parseJsFile)
+    : parseJsFile(readFile.sync(filepath));
```

Comment:
I like this method of creating the `parse...` functions and only differentiating sync and async in these few lines!